suite/tests/client-interface/mangle_suspend.c

```diff
@@ -34,6 +34,11 @@
  * by asynch interrupt.
  */
 
+/* clang-format off */
+/* XXX: clang-format incorrectly detected a tab difference at "clang-format on"
```

Comment:
A "tab difference"?  As in, there are tab characters somewhere?  This disabling of clang-format for all the code here seems like overkill.  Are you sure there's not some other solution?  I don't really understand the problem though.  Do you mean clang-format-diff?

Reply:
It's the same issue as a while ago in drreg-test.c. You actually wanted to 'clang-format off' the entire file back then per-review. I'll re-phrase the comment, it's not really a 'tab difference' per se.

Reply:
I see, ok.